### PR TITLE
LGA-882 End page with postcode check

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

Updates most elements on the legal-adviser page.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/checker/result/_find-legal-adviser.html [hunk 1]
```diff
@@ -3,39 +3,70 @@
...
+  
+  {% if request and request.args and request.args.get('postcode') and request.args.get('postcode')|length > 2 and request.args.get('postcode')|length < 9 %}
+    {% set scot_postcode = "AB DD DG EH FK G1 G2 G3 G4 G5 G6 G7 G8 G9 G0 HS IV KA KW KY ML PA PH TD ZE" %}
+    {% set ni_postcode = "BT" %}
+    {% if request.args.get('postcode')[:2]|upper in scot_postcode 
+    or request.args.get('postcode')[:2]|upper in ni_postcode
+    or request.args.get('postcode')[:2]|upper == "IM" 
+    or request.args.get('postcode')[:2]|upper == "GY" 
+    or request.args.get('postcode')[:2]|upper == "JE"
+    %}
```
Comment (said-moj) [inline cla_public/templates/checker/result/_find-legal-adviser.html]: This seems a bit hard to follow in the template

What do you think of having these in is_ variables (is_scottish_postcode, is_ni_postcode, is_jersey_postcode...) and moving the logic to determine them in the handler

Comment (MalcolmVonMoJ) [reply cla_public/templates/checker/result/_find-legal-adviser.html]: I agree.  I'll change this.
